homeassistant/components/denon/media_player.py

```diff
@@ -257,11 +257,11 @@ def volume_down(self):
 
     def set_volume_level(self, volume):
         """Set volume level, range 0..1."""
-        self.telnet_command("MV" + str(round(volume * self._volume_max)).zfill(2))
+        self.telnet_command(f"MV{str(round(volume * self._volume_max)).zfill(2)}")
 
     def mute_volume(self, mute):
         """Mute (true) or unmute (false) media player."""
-        self.telnet_command("MU" + ("ON" if mute else "OFF"))
+        self.telnet_command(f"MU{'ON' if mute else 'OFF'})")
```

Comment:
Maybe move the if statement outside of the fstring?

Reply:
Yeah sounds good.